src/plugins/intel_cpu/src/utils/ngraph_transformation.hpp

```diff
@@ -76,9 +77,8 @@ class TransformationDumper {
 
         if (config.dumpIR.format.filter[DebugCapsConfig::IrFormatFilter::Xml]) {
             std::string xmlFile(pathAndName + ".xml");
-            std::string binFile("/dev/null");  // @todo make it crossplatform using dummy implementation of std::ostream
 
-            serializer.register_pass<ov::pass::Serialize>(xmlFile, binFile);
+            serializer.register_pass<ov::pass::Serialize>(xmlFile, NULL_STREAM);
```

Comment:
Sounds like it worth to make `binPath` argument optional in the ov::pass::Serialize pass itself. Anyway not blocking this PR.

Reply:
Actually, we need to pass "/dev/null" or "NUL" there, but if binary file is not specified the behavior will be different: it will open bin file anyway